### PR TITLE
refactor: create `args` folder

### PR DESCRIPTION
**Pull request by dsherret**
Moves the following files to cli/args:

* cli/config_file.rs
* cli/flags.rs
* cli/flags_allow_net.rs

Main `cli` folder is starting to get busy and will get more busy after an upcomming refactor I'm going to do which will create a structure that contains all the arguments (CLI flags and config file).

### CODE REVIEW

File: cli/args/mod.rs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
```
Comment (dsherret) [inline cli/args/mod.rs:1]: I'll create a new struct here in a future PR that will have the config file and flags in it.

File: cli/args/mod.rs [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+pub mod config_file;
+pub mod flags;
+
+mod flags_allow_net;
+
+pub use config_file::*;
+pub use flags::*;
```
Comment (bartlomieju) [inline cli/args/mod.rs:9]: Can we get rid of wildcards here and explicitly list exports?

**Review by bartlomieju — APPROVED** [review]
LGTM

File: cli/args/mod.rs (hunk 2 above)
Comment (dsherret) [reply cli/args/mod.rs:9]: Why not use wildcards?

Comment (dsherret) [reply cli/args/mod.rs:9]: I'm planning to limit the config_file exports to reduce what's visible from this module in the next PR, but I think it's a lot easier to just do `pub use flags::*` for the flags.

Comment (bartlomieju) [reply cli/args/mod.rs:9]: Okay sure

**Pull request merged**